pkg/virtualKubelet/apiReflection/reflectors/blacklists.go

```diff
@@ -0,0 +1,17 @@
+package reflectors
+
+import (
+	apimgmt "github.com/liqotech/liqo/pkg/virtualKubelet/apiReflection"
+)
+
+type blackList map[string]struct{}
+
```

Comment:
I think a comment to this function would help to understand how the blacklist works.